### ISSUE TITLE
Docstring popup not showing correctly ?

### ISSUE THREAD

**Issue by houstonian** [DamnWidget/anaconda#689]
## Expected Behaviour
Docstring popup
<img width="776" alt="screenshot 2017-08-12 20 20 52" src="https://user-images.githubusercontent.com/18011051/29246184-ff95038a-7fbf-11e7-8459-8ab7679b3f0d.png">

## Actual Behaviour
<img width="381" alt="screenshot 2017-08-12 20 15 09" src="https://user-images.githubusercontent.com/18011051/29246185-0f98c406-7fc0-11e7-8ef5-be3515681f67.png">

No Docstring on multiple functions such as abs() which I used for a test example.

## Steps to Reproduce
Type abs() - no docstring popup shows up

## ST3, Anaconda and OS versions
Sublime Text 3, Build 3126
Anaconda v2.1.17
OS X Sierra 10.12.6 

## ST3 Console Logs (this log is when doing a build with the Anaconda Builder. It seems like the problem lies in that it is not recognizing the python 3 interpreter even when full path is provided)
as I did:

> {
> 	"python_interpreter": "/usr/local/bin/python3",
> }

/usr/local/opt/python3/bin/python3.6: can't find '__main__' module in ''
[Finished in 0.1s with exit code 1]
[shell_cmd: "/usr/local/bin/python3" -u ""]
[dir: /Applications/Sublime Text.app/Contents/MacOS]
[path: /usr/bin:/bin:/usr/sbin:/sbin]

## Anaconda's JsonServer Logs
**Note**: Anaconda's JsonServer logs can be found in:
* Linux: ~/.local/share/anaconda/logs
* OS X: ~/Library/Logs/anaconda
* Windows: %APPDATA\\Anaconda\\Logs

**Comment by dbader** [comment]
I'm getting the same result. Docstrings appear fine for non-builtins, e.g. `logging.info`:

<img width="722" alt="screenshot 2017-08-14 09 06 25" src="https://user-images.githubusercontent.com/306708/29280396-eed69f98-80cf-11e7-9d93-84d85b074848.png">

```
2017-08-14 09:06:14,835: INFO    : client requests: doc
2017-08-14 09:06:14,840: INFO    : About push back to ST3: b'{"success": true, "doc": "logging.info\\ninfo(msg, *args, **kwargs)<br><br>Log a message with severity \'INFO\' on the root logger. If the logger has<br>no handlers, call basicConfig() to add a console handler with a pre-defined<br>format.", "uid": "e33f3720179f4c53abbceb81952954fd"}\r\n'
2017-08-14 09:06:16,029: INFO    : client requests: doc
2017-08-14 09:06:16,036: INFO    : About push back to ST3: b'{"success": true, "doc": "logging.info\\ninfo(msg, *args, **kwargs)<br><br>Log a message with severity \'INFO\' on the root logger. If the logger has<br>no handlers, call basicConfig() to add a console handler with a pre-defined<br>format.", "uid": "5e23fa47e5484600bec0cd0cc0c60cc9"}\r\n'
2017-08-14 09:06:17,075: INFO    : client requests: doc
2017-08-14 09:06:17,082: INFO    : About push back to ST3: b'{"success": true, "doc": "logging.info\\ninfo(msg, *args, **kwargs)<br><br>Log a message with severity \'INFO\' on the root logger. If the logger has<br>no handlers, call basicConfig() to add a console handler with a pre-defined<br>format.", "uid": "e2eab47c5a29472e9732ce17d1b03abd"}\r\n'
2017-08-14 09:06:18,336: INFO    : client requests: doc
2017-08-14 09:06:18,344: INFO    : About push back to ST3: b'{"success": true, "doc": "logging.info\\ninfo(msg, *args, **kwargs)<br><br>Log a message with severity \'INFO\' on the root logger. If the logger has<br>no handlers, call basicConfig() to add a console handler with a pre-defined<br>format.", "uid": "3ca061842f9b444eb2883c78389576e9"}\r\n'
```

But it seems like the results returned by the JsonServer are lacking docstrings for builtins like `print` or `abs`:

<img width="317" alt="screenshot 2017-08-14 09 08 03" src="https://user-images.githubusercontent.com/306708/29280467-31f3189c-80d0-11e7-9a3d-33854f84514b.png">

```
2017-08-14 09:07:57,855: INFO    : client requests: doc
2017-08-14 09:07:57,862: INFO    : About push back to ST3: b'{"success": true, "doc": "print\\n", "uid": "69dcbdc368c24071917bc17c8b688208"}\r\n'
```

I'm on Python 3.6.2 installed via Homebrew:

```
{
    "python_interpreter": "/usr/local/bin/python3",
    "anaconda_linting": false,
    "pep8": false
}
```

But getting the same result on Python 2.7.10 that ships with macOS Sierra.

Thanks for the fantastic package and all of your hard work :)

**Comment by DamnWidget** [comment]
This should be fixed in v2.1.21

**Comment by dbader** [comment]
Woohoo, works great again on 2.1.21! Thanks so much, Oscar 😄 

<img width="480" alt="screenshot 2017-08-18 08 12 28" src="https://user-images.githubusercontent.com/306708/29465127-ffee3fd2-83ec-11e7-8e78-f3e64a58dfa8.png">
